ui/css/module/_listitem.scss

```diff
@@ -10,4 +10,19 @@
   .list-item-text {
     @include paragraph-with-marker();
   }
+
+  .node-paragraph {
+    margin-bottom: 0;
+  }
+}
+
+.node-paragraph {
```

Comment:
I don't understand the intent here -- why do we want different line heights for lists inside paragraphs?

Reply:
If the line-height of the `list-item-marker` and the content inside `list-item-text` are different, then we end up with the unaligned baseline problem. If the `node-paragraph` has a larger/smaller line-height, the `marker` gets thrown off. Do you have any thoughts on a good way to resolve this?

Reply:
The basic problem is that we'd like to have each of these node types be compose-able -- with some exception, any type of node can be inside any other type. Given that, any time we're testing for nested css classes, we're in a bad situation.

Would a potential solution be to remove top-margins and _only_ add bottom margins? We probably shouldn't have padding as that will be additive rather than collapsing. Does that make sense?

Reply:
I agree about the nested nodes.

I think Im missing something around the top-margin changes.

Reply:
Apologies around top-margin -- I was poorly multi-tasking and thought we were talking about paragraphs aligning too low due to a top margin. Let's try a better question: should the line-heights of markers and paragraphs be different?